build/azure-pipelines/sql-product-build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
   timeoutInMinutes: 60
 
 - job: Linux
-  condition: and(succeeded(), eq(variables['VSCODE_BUILD_LINUX'], 'true'))
+  condition: and(succeeded(), eq(variables['VSCODE_BUILD_LINUX'], 'true'), ne(variables['VSCODE_QUALITY'], 'saw'))
```

Comment:
So I think for now you should include the linux builds too since that's the only place we run the unit tests and that would be good to have to make sure the saw builds aren't broken in some weird way.

Reply:
reverted